examples/bare-rn/App.tsx

```
@@ -73,7 +73,7 @@ function initHelium(onEvent: (event: HeliumPaywallEvent) => void) {
   if (isIOS) {
     // Must be called before initialize(). Omitting paymentProcessors enables all
     // (Paddle + Stripe), so any web-checkout paywall in the org can show.
-    enableExternalWebCheckout({successURL: RETURN_URL, cancelURL: RETURN_URL});
+    enableExternalWebCheckout({redirectURL: RETURN_URL, paymentProcessors: ['paddle', 'stripe']});
```

ESLint

[error] Replace `redirectURL:·RETURN_URL,·paymentProcessors:·['paddle',·'stripe']` with `⏎······redirectURL:·RETURN_URL,⏎······paymentProcessors:·['paddle',·'stripe'],⏎····`

(prettier/prettier)